conjureup/controllers/newcloud/common.py

```diff
@@ -52,9 +52,9 @@ def try_get_creds(cloud):
         return existing_creds['credentials'][cloud]['default-credential']
 
     # XXX we should really prompt to select because this is non-deterministic
```

Comment:
I think this should be fixed in a future PR that adds the ability for users to select from a list of credentials for a cloud.